scripts/create_nf_lib.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+# Convert the icons.tsv file's data to Nerd Font lib's format
+# to update their lib file with the new glyphs.
+# Usage: ./create_lib.sh
+# 	Copy missing glyphs from the output to the
+# 	<PATH>/nerd-fonts/bin/scripts/lib/i_logos.sh file.
+
+# Get script directory to set file path relative to it
+file_path="$(
+	cd -- "$(dirname "${0}")" >/dev/null 2>&1 || exit
+	pwd -P
+)/../icons.tsv"
+
+# Read the tsv file excluding the header
+sed '1d' "${file_path}" | while IFS=$'\t' read -r offset _ classname; do
+	# Calculate the codepoint, font-logos starts at 0xF300
+	codepoint=$((62208 + offset))
+	# Get the glyph from the codepoint
+	glyph=$(printf "\\U%x" "${codepoint}" 2>/dev/null)
```

Comment:
I believe this must be lower-case `u`, the codepoints here are all 4 digit.
At least it will depend on the `printf` implementation and how forgiving it is.

Can not really check because I'm on MacOS right now where the script does not work at all ;)

<img width="1116" alt="Screenshot 2024-09-14 at 09 38 55" src="https://github.com/user-attachments/assets/09696656-5fb9-4e44-8352-3430ce4182ac">

Reply:
You're correct